django/templatetags/tz.py

```diff
@@ -1,13 +1,35 @@
 from datetime import datetime, tzinfo
 
-import pytz
+try:
+    import zoneinfo
+except ImportError:
+    from backports import zoneinfo
 
+from django.conf import settings
 from django.template import Library, Node, TemplateSyntaxError
 from django.utils import timezone
 
 register = Library()
 
 
+class UnknownTimezoneException(BaseException):
+    pass
+
+
+# RemovedInDjango50Warning
+def timezone_constructor(tzname):
+    if settings.USE_DEPRECATED_PYTZ:
+        import pytz
```

Comment:
I suspect I'm wrong, but... I don't think there is test coverage for this branch here? That is, there is coverage for the `zoneinfo` branch by not the `pytz` one.